bin/millau/runtime/src/lib.rs

```diff
@@ -596,6 +596,29 @@ impl_runtime_apis! {
 	}
 }
 
+/// Proof Rialto account ownership from Millau.
+///
+/// The byte vector returned by this function should be signed with a Rialto account private key.
+/// This way, the owner of `millau_account_id` on Millau proves that the Rialto account private key
+/// is also under his control.
+pub fn rialto_account_ownership_proof<Call, AccountId, SpecVersion>(
```

Comment:
This technically isn't really a proof, it just returns a payload that should be signed by the target account (and the signature + payload + public address is a proof). But it's good enough.

Reply:
How should this be named then?

Reply:
I'd change `call_dispatch::account_ownership_proof -> call_dispatch::target_origin_signed_payload`
and `rialto_account_ownership_proof -> rialto_origin_signed_payload`

but I don't think these names are that much better, hence I didn't even propose that.

Reply:
How about `account_ownership_digest`?

Reply:
works for me!

Reply:
Changed the names. Will merge, once CI is green.